actions/run_action.go

```diff
@@ -120,7 +120,7 @@ func (run *RunAction) doRun(context debos.DebosContext) error {
 		}
 		cmdline = []string{run.scriptPath}
 		cmdline = append(cmdline, run.scriptArgs...)
-		label = path.Base(run.Script)
+		label = path.Base(run.scriptPath)
```

Comment:
but then you don't see the arguments to the script anymore, which is a loss.. Probably this doesn't belong in this PR (it doesn't help with your stated target i think) but preventing "pollution" we could do either by:
* unify truncating the length like the Command handling does
* printing what gets run *first* and then either not or only providing a minimal label on each line after